src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -27,4 +27,51 @@
   @JsonSubTypes.Type(value = GeneralCommand.class),
   @JsonSubTypes.Type(value = CollectionCommand.class),
 })
-public interface Command {}
+public interface Command {
+
+  /**
+   * commandName that refers to the public command name
+   *
+   * <p>e.g. FindKeyspacesCommand publicCommandName -> findKeyspaces. CreateCollectionCommand
+   * publicCommandName -> createCollection
+   */
+  PublicCommandName publicCommandName();
+
+  /** Enum class for API public command name This is what user uses for command json body. */
```

Comment:
nit: missing "." -> "... command name. This is... "

Reply:
thanks